File: Assets/scenes/landinglaunchscene/script/HyperspaceStream.js
#pragma strict

var rotAmt : Vector3;

private var orgRot : Quaternion;


function Start () {
	orgRot = transform.rotation;
}

function Update () {
	var xAng : float = Mathf.Sin(Time.fixedTime) / rotAmt.x;
	var yAng : float = Mathf.Cos(Time.fixedTime) / rotAmt.y;
	transform.rotation = orgRot * Quaternion.EulerAngles(xAng,yAng,0);
	

}

File: Assets/scenes/landinglaunchscene/script/HyperspaceStream.cs
using UnityEngine;
using System;


public class HyperspaceStream:MonoBehaviour{
	
	public Vector3 rotAmt;
	
	Quaternion orgRot;
	
	
	public void Start() {
		orgRot = transform.rotation;
	}
	
	public void Update() {
		float xAng = Mathf.Sin(Time.fixedTime) / rotAmt.x;
		float yAng = Mathf.Cos(Time.fixedTime) / rotAmt.y;
		transform.rotation = orgRot * Quaternion.EulerAngles(xAng,yAng,0.0f);
		
	
	}
}